python/cufinufft/cufinufft/_cufinufft.py

```diff
@@ -116,14 +116,14 @@ class NufftOpts(ctypes.Structure):
 
 _set_pts = lib.cufinufft_setpts
 _set_pts.argtypes = [
-    c_void_p, c_int64, c_void_p, c_void_p, c_void_p, ctypes.c_int, c_double_p,
-    c_double_p, c_double_p]
+    c_void_p, c_int64, c_void_p, c_void_p, c_void_p, ctypes.c_int64, c_void_p,
+    c_void_p, c_void_p]
```

Comment:
May I know why this change?

Reply:
Those arguments weren't used previously since we didn't exercise the type 3 interface (they are used to set the target points). This lines them up with the other four arguments (for the source points). I don't recall if it threw an error, but at any rate, it's probably good to have the right signature here.